css/layout-css/small-card-style.upload.css

```diff
@@ -18,6 +18,8 @@
   background-color: transparent;
   transform: translate3d(100%, 0, 0);
   transition: all 0.2s ease-out;
+  height: 100vh;
```

Comment:
If the height is now statically set to `100vh` with a padding at the bottom, then does this mean `bottom: 0` is no longer necessary? If so, please remove it.

Reply:
Yes, thank you, I removed it.